superset/datasets/schemas.py

```diff
@@ -34,14 +34,14 @@ def validate_python_date_format(value: str) -> None:
         r"""
         ^(
             epoch_s|epoch_ms|
-            (?P<date>%Y(-%m(-%d)?)?)([\sT](?P<time>%H(:%M(:%S(\.%f)?)?)?))?
+            (?P<date>%Y([-/]%m([-/]%d)?)?)([\sT](?P<time>%H(:%M(:%S(\.%f)?)?)?))?
         )$
         """,
         re.VERBOSE,
     )
     match = regex.match(value or "")
     if not match:
-        raise ValidationError(_("Invalid date/timestamp format"))
+        raise ValidationError([_("Invalid date/timestamp format")])
```

Comment:
The API error response was malformed without this, was giving `["I", "n", "v", ...]`. This fixes it